src/Private/Cryptography.ps1

```diff
@@ -363,6 +510,8 @@ function ConvertTo-PodeBase64UrlValue {
 }
 
 function ConvertFrom-PodeJwtBase64Value {
+    [CmdletBinding()]
+    [OutputType([string])]
```

Comment:
This should actually be `[pscustomobject]`, due to the `ConvertFrom-Json` on the `return`

Reply:
you are right I messup here